src/vm/runners/cairo_runner.rs

```diff
@@ -360,6 +360,20 @@
         for (_, builtin) in vm.builtin_runners.iter() {
             builtin.add_validation_rule(&mut vm.memory)?;
         }
+
+        // Mark all addresses from the program segment as accessed
+        let mut initial_accessed_addresses: Vec<Relocatable> = Vec::new();
+        let prog_segment_index = self
+            .program_base
+            .as_ref()
+            .unwrap_or(&Relocatable::from((0, 0)))
+            .segment_index;
+
+        for offset in 0..self.program.data.len() {
+            initial_accessed_addresses.push(Relocatable::from((prog_segment_index, offset)));
+        }
```

Comment:
This looks like something that can be done with `.collect()` instead.

Reply:
done!